nox/_options.py

```diff
@@ -225,19 +233,43 @@ def _posargs_finalizer(
     return posargs[dash_index + 1 :]
 
 
+def _python_completer(
+    prefix: str, parsed_args: argparse.Namespace, **kwargs: Any
+) -> Iterable[str]:
+    global_config = parsed_args
+    module = load_nox_module(global_config)
+    manifest = discover_manifest(module, global_config)
+    return (
```

Comment:
Can we use `filter` here?

Like this 

```python3
return filter(None, (session.func.python for session, _ in manifest.list_all_sessions()))
```
It can be slightly faster and is more readable. It also scales better when there are more complex filtering conditions.

Reply:
49634fcb169165f1a822a97272ee6628d258e553